src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts

```diff
@@ -139,11 +140,31 @@ export class HearingActualAddEditSummaryComponent implements OnInit, OnDestroy {
       ? true : false;
   }
 
-  private isHearingActualsPartiesAvailable(hearingActualsMainModel: HearingActualsMainModel) {
-    return hearingActualsMainModel.hearingActuals && hearingActualsMainModel.hearingActuals.actualHearingDays && hearingActualsMainModel.hearingActuals.actualHearingDays.length > 0 &&
-      hearingActualsMainModel.hearingActuals.actualHearingDays && hearingActualsMainModel.hearingActuals.actualHearingDays[0].actualDayParties &&
-      hearingActualsMainModel.hearingActuals.actualHearingDays && hearingActualsMainModel.hearingActuals.actualHearingDays[0].actualDayParties.length > 0
-      ? true : false;
+  private isHearingActualsPartiesAvailable(hearingActualsMainModel: HearingActualsMainModel): boolean {
+    let isActualPartiesAvailable: boolean = true;
+    const isActualHearingDaysAvailable = hearingActualsMainModel.hearingActuals && hearingActualsMainModel.hearingActuals.actualHearingDays && hearingActualsMainModel.hearingActuals.actualHearingDays.length > 0;
+    if (isActualHearingDaysAvailable) {
+      this.actualHearingDays.forEach((actualHearingDay) => {
+        const isDetailsProvided = this.getStatusLabel(actualHearingDay);
```

Comment:
should getStatusLabel function be renamed (maybe isDetailedProvidedForDay) ? seems confusing otherwise